lib/Chai.d.ts

```diff
@@ -7,6 +7,28 @@ import {Should} from './Should';
 import {Config} from './Config';
 import {Utils} from './Utils';
 
+
+export * from './Assertion';
+export * from 'assertion-error';
+export {AssertStatic} from './Assert';
+export {AssertionStatic, Assertion} from './Assertion';
+export {ExpectStatic} from './Expect';
+export {Should} from './Should';
+export {Config} from './Config';
+export {Utils} from './Utils';
+
+export declare var expect:ExpectStatic;
+export declare var assert:AssertStatic;
+export declare var config:Config;
+export declare var util:Utils;
+export function should(): Should;
+export function use(fn: ((chai: any, utils: Utils) => void) | ((chai: any) => void) ): Chai;
```

Comment:
`use (fn: ChaiExtension): Chai`? I'd also move this to the main export to be able to kill the `Chai` name. It'll then look like `import * as Chai from './lib/Chai'`.